### DISCUSSION TITLE
Code-behind file location control

### DISCUSSION THREAD

**Discussion by aguzev** [reqnroll/Reqnroll#285] [Q&A] [answered]
Are code-behind file locations just defined as [CodeBehindFile item property in the .props file](https://github.com/reqnroll/Reqnroll/blob/311912a7e980d182597550cb9c246617d4a7afbb/Reqnroll.Tools.MsBuild.Generation/build/Reqnroll.Tools.MsBuild.Generation.props#L53) or assumed somewhere else in code? In other words, is changing the item property definition enough to relocate generated output to obj/*.feature.cs?

**Comment by gasparnagy** [comment, marked as answer]
@aguzev No, unfortunately the location of these cannot be changed with a simple setting like that currently. We currently focus on #11 , so the code behind files will not be needed as they are now.
